util/blockbreaker.py

```diff
@@ -130,6 +130,9 @@ def json_to_stream(json_obj, cfg, idx):
             endpoint_type = json_blk["type"]
             if not validate_schema(json_blk, "schema-" + endpoint_type + ".json"):
                 return None
+            if endpoint_type == "remotehosts":
+                stream = endpoint_type + ",run-file=" + args.json_file + ",endpoint-index=" + str(idx)
```

Comment:
so the endpoint_type is not part of the run-file ? why still need to add it here ?

Reply:
This "stream" is being sent to `rickshaw-run` -- it uses the first part (the `endpoint_type`) to know which endpoint to invoke and then the rest is sent do the endpoint itself.